new for configs (#965)
vesoft-inc/nebula-docs: docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md

## Configuration files

> ### Configuration files for clusters installed from source, with an RPM/DEB package, or a TAR package

Reviewer: what does "installed from souce" mean? the source code?

Author: yup, means the source code

> Nebula Graph provides two initial configuration files for each service, `<service_name>.conf.default` and `<service_name>.conf.production`. You can use them in different scenarios conveniently. The default path is `/usr/local/nebula/etc/` for clusters installed from source and with a RPM/DEB package; the path is `<install_path>/<tar_package_directory>/etc` for clusters installed with a TAR package.

Reviewer: ```suggestion
Nebula Graph provides two initial configuration files for each service, `<service_name>.conf.default` and `<service_name>.conf.production`. You can use them in different scenarios conveniently. For clusters installed from source and with a RPM/DEB package, the default path is `/usr/local/nebula/etc/`. For clusters installed with a TAR package, the path is `<install_path>/<tar_package_directory>/etc`.
```
we seldom use semicolon in docs as i recall

    It is not recommended to modify the value of `local_config` to `false`. If modified, the Nebula Graph service will first read the cached configurations, which may cause configuration inconsistencies between clusters and cause unknown risks.

### Configuration files for clusters installed with Docker Compose

> For clusters installed with Docker Compose, the configuration file's default installation path of the cluster is `<install_path>/nebula-docker-compose/docker-compose.yaml`. The parameters in the `command` field of the file are the start parameters for each service.  

Reviewer: "start parameters" looks a bit strange here.

Author: > 

has the same meaning with Launch Parameters. will use launch instead

### Configuration files for clusters installed with Nebula Operator

For clusters installed with Kubectl through Nebula Operator, the configuration file's path is the path of the cluster YAML file. You can modify the configuration of each service through the `spec.{graphd|storaged|metad}.config` parameter.  

!!! note

    The services cannot be configured for clusters installed with Helm.

## Modify configurations

> By default, each Nebula Graph service gets configurations from its configuration files. You can modify configurations and make them valid according to the following steps:

Reviewer: ```suggestion
By default, each Nebula Graph service gets configured from its configuration files. You can modify configurations and make them valid according to the following steps:
```

* For clusters installed from source, with a RPM/DEB, or a TAR package

  1. Use a text editor to modify the configuration files of the target service and save the modification.

  2. Choose an appropriate time to restart **all** Nebula Graph services to make the modifications valid.

* For clusters installed with Docker Compose

  1. In the `<install_path>/nebula-docker-compose/docker-compose.yaml` file, modify the configurations of the target service.

>   2. In the `nebula-docker-compose` directory, run the command `docker-compose up -d` to restart the configuration-changed service.

Reviewer: ```suggestion
  2. In the `nebula-docker-compose` directory, run the command `docker-compose up -d` to restart the service involving configuration modifications.
```
